lib/expression/v2/compile.ex

```diff
@@ -178,11 +169,7 @@ defmodule Expression.V2.Compile do
   end
 
   defp quoted(atom) when is_binary(atom) do
-    {{:., [], [{:__aliases__, [alias: false], [:Map]}, :get]}, [],
-     [
-       {{:., [], [{:context, [], nil}, :vars]}, [no_parens: true], []},
-       atom
-     ]}
+    {{:., [], [Access, :get]}, [], [context_dot_vars(), atom]}
```

Comment:
again `context.vars["atom"]` ended up being more readable than lots of nested `Map.get` calls